tests/order_modifiers_integration/Admin/Editor_Test.php

```diff
@@ -256,4 +261,13 @@ function () {
 
 		$this->assertEquals( $fixture(), $should_enqueue_assets() );
 	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_not_register_with_classy_editor() {
+		$this->set_fn_return( 'tec_using_classy_editor', true );
```

Comment:
You can use the `tec_using_classy_editor` filter to control this, no need to use the `uopz` mock.